coala_quickstart/info_extraction/Info.py

```diff
@@ -0,0 +1,34 @@
+class Info:
+    description = 'Some information'
+    value_type = (object,)
+
+    def __init__(self,
+                 source,
+                 value,
+                 extractor=None,
+                 **kwargs):
+        """
+        :param source:    Source from where the information is discovered.
+        :param value:     Value of the infomation to be represented.
+        :param extractor: ``InfoExtractor`` instance used to extract the
+                          information.
+        """
+        if not self.value_type:
+            self.value_type = (object,)
+
+        if not isinstance(value, self.value_type):
+            raise TypeError("The value {} does not match the allowed "
+                            "value types {}".format(value, self.value_type))
+        self.source = source
+        self._value = value
+        self.extractor = extractor
+        for key, val in kwargs.items():
+            setattr(self, key, val)
+
+    @property
+    def value(self):
+        return self._value
```

Comment:
Agreed. The reasoning for me was if in future we want to have different internal representation and different output representation, then we just need to make changes just in the getter function.